egs/wsj/s5/steps/nnet3/chain/build_tree.sh

```diff
@@ -24,7 +24,7 @@ cluster_thresh=-1  # for build-tree control final bottom-up clustering of leaves
 frame_subsampling_factor=1
 leftmost_questions_truncate=10
 tree_stats_opts=
-cluster_phones_opts=
+cluster_phones_opts="--pdf-class-list=0"
```

Comment:
Was it necessary for some reason to change this?
This will have the effect that for finding the phone classes, it just uses the 1st transition of the phone, rather than the self-loop.  it's not clear to me that this would be better.
Actually, I can see why this would have been necessary when you had that bug and the tree was not being built right, but this is probably not needed now.